SPECS/util-linux/util-linux.spec

```diff
@@ -18,6 +19,7 @@ BuildRequires:  pam-devel
 Requires:       %{name}-libs = %{version}-%{release}
 Requires:       audit-libs
 Conflicts:      toybox
+Conflicts:      shadow-utils < 4.9-10
```

Comment:
Nit: Could use a comment explaining the conflict

Reply:
Sure, updated